lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/schema/impl/SchemaContextHolderImpl.java

```diff
@@ -177,7 +177,7 @@ private Set<GnmiYangModel> readDependencyModels(final YangModelDependencyInfo de
         for (ModuleImport moduleImport : dependencyInfo.getDependencies()) {
             if (!processedModuleNames.contains(moduleImport.getModuleName())) {
                 final GnmiDeviceCapability importedCapability = new GnmiDeviceCapability(moduleImport.getModuleName(),
```

Comment:
Rework `GnmiDeviceCapability` class in separate commit to do not carry on `semanticVersion`.

Reply:
This is not possible, because we use it to store SemVer from capabilities response:
https://github.com/PANTHEONtech/lighty/blob/master/lighty-modules/lighty-gnmi/lighty-gnmi-sb/src/main/java/io/lighty/gnmi/southbound/requests/utils/GnmiRequestUtils.java#L34